examples/ez-food/menu-svc/service.bal

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+import ballerina/http;
+import ballerina/log;
+
+type MenuItem record {|
+    int id;
+    string name;
+    decimal price;
+    string description;
+    int preparationTimeMinutes;
+|};
+
+configurable MenuItem[] menu = ?;
```

Comment:
any particular reason to make this configurable?

Reply:
By making this configurable the sample menu-items were separated out into the `Config.toml` and in return the resulting ballerina file is more concise and readable.